### PR TITLE
Wincatalog.GetColorXP version 1.01

### PR DESCRIPTION
**Pull request by tbringuier**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/14386)

### CODE REVIEW

File: manifests/w/Wincatalog/GetColorXP/1.01/Wincatalog.GetColorXP.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Wincatalog.GetColorXP
+PackageVersion: 1.01
+PackageLocale: en-US
+Publisher: WinCatalog
```
Comment (ItzLevvie) [inline manifests/w/Wincatalog/GetColorXP/1.01/Wincatalog.GetColorXP.locale.en-US.yaml:6]:
```suggestion
Publisher: Mar Software
```

File: manifests/w/Wincatalog/GetColorXP/1.01/Wincatalog.GetColorXP.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Wincatalog.GetColorXP
+PackageVersion: 1.01
+PackageLocale: en-US
+Publisher: WinCatalog
+PackageName: GetColor
```
Comment (ItzLevvie) [inline manifests/w/Wincatalog/GetColorXP/1.01/Wincatalog.GetColorXP.locale.en-US.yaml:7]:
```suggestion
PackageName: GetColor!
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/w/Wincatalog/GetColorXP/1.01/Wincatalog.GetColorXP.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Wincatalog.GetColorXP
```
Comment (ItzLevvie) [inline manifests/w/Wincatalog/GetColorXP/1.01/Wincatalog.GetColorXP.locale.en-US.yaml:3]:
```suggestion
PackageIdentifier: WinCatalog.GetColor
```

File: manifests/w/Wincatalog/GetColorXP/1.01/Wincatalog.GetColorXP.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: Wincatalog.GetColorXP
```
Comment (ItzLevvie) [inline manifests/w/Wincatalog/GetColorXP/1.01/Wincatalog.GetColorXP.yaml:3]:
```suggestion
PackageIdentifier: WinCatalog.GetColor
```

File: manifests/w/Wincatalog/GetColorXP/1.01/Wincatalog.GetColorXP.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Wincatalog.GetColorXP
```
Comment (ItzLevvie) [inline manifests/w/Wincatalog/GetColorXP/1.01/Wincatalog.GetColorXP.installer.yaml:3]:
```suggestion
PackageIdentifier: WinCatalog.GetColor
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/w/Wincatalog/GetColorXP/1.01/Wincatalog.GetColorXP.locale.en-US.yaml (hunk 1 above)
Comment (ItzLevvie) [inline manifests/w/Wincatalog/GetColorXP/1.01/Wincatalog.GetColorXP.locale.en-US.yaml:6]:
```suggestion
Publisher: Mar Software
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by MichelBaie** [comment]
Thanks !

**Comment by ItzLevvie** [comment]
You'll also need to change the folder name and the file name to match WinCatalog.GetColor

i.e. rename `Wincatalog` to `WinCatalog` and rename `GetColorXP` to `GetColor`.

**Pull request closed without merging**